docs/sources/flow/reference/components/loki.source.kubernetes_events.md

```diff
@@ -26,11 +26,12 @@ log entries to the list of receivers passed in `forward_to`.
 
 `loki.source.kubernetes_events` supports the following arguments:
 
-Name | Type | Description | Default | Required
----- | ---- | ----------- | ------- | --------
+Name | Type | Description                                      | Default | Required
+---- | ---- |--------------------------------------------------| ------- | --------
 `job_name` | `string` | Value to use for `job` label for generated logs. | `"loki.source.kubernetes_events"` | no
-`namespaces` | `list(string)` | Namespaces to watch for Events in. | `[]` | no
-`forward_to` | `list(LogsReceiver)` | List of receivers to send log entries to. | | yes
+`log_format` | `string` | Log format, either "logfmt" or "json". | `"logfmt"` | no
```

Comment:
I'd personally leave minimal description (like "Format of the log") and then below the table explain a bit more in detail the function of this field and the possible values that it takes. You can reuse the text from `eventhandler-config.md`